### PR TITLE
[Spotify] Add filler image

### LINKED ISSUE #60: [ Suggestion ] [ Spotify ] Add some kind of filler image when playing local songs
The spotify modal looks kind of weird when you play a local file due to the huge empty space created by the fact that there is no thumbnail.

### PR DESCRIPTION
**Pull request by davidcralph**
Closes #60

### CODE REVIEW

**Comment by davidcralph** [comment]
works very well yes

File: src/Powercord/plugins/pc-spotify/Modal/Modal.jsx [hunk 1]
```diff
@@ -89,11 +89,12 @@ module.exports = class Modal extends React.Component {
 
   updateData (playerState) {
     if (playerState && playerState.currently_playing_type === 'track') {
-      return this.setState({
+      return powercord.pluginManager.get('pc-spotify').getSpotifyLogo().then(value => {
```
Comment (cyyynthia) [inline src/Powercord/plugins/pc-spotify/Modal/Modal.jsx]: I'm not a fan of fetching the logo everytime the player state is updated. What you should to is save that in an object property, and re-use it if the img is `null`

**Review by cyyynthia — CHANGES_REQUESTED** [review]

File: src/Powercord/plugins/pc-spotify/Modal/Modal.jsx (hunk 1 above)
Comment (davidcralph) [reply src/Powercord/plugins/pc-spotify/Modal/Modal.jsx]: yikes i don't get that my JS knowledge is failing me 2.0

**Comment by GriefMoDz** [comment]
In reference to @Bowser65's requested changes, here's how I would go about in assigning the logo to an object property and also re-use it's value if `currentItem.img` is `null`: [click me!](https://gist.github.com/GriefMoDz/09c06acc89c6b3d52b8e8ea93269006c)

**Note**: As opposed to linking a gist link, I wasn't able to request my own changes due insufficient permissions (at least not on other lines) so I apologise if this comes across as an inconvience.

**Comment by davidcralph** [comment]
Implemented that gist, thanks! 👍

**Review by cyyynthia — APPROVED** [review]

**Review by aetheryx — APPROVED** [review]

**Pull request merged**